torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3228,6 +3232,26 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard',
                                 device_type='cpu', dtypes=[torch.complex64])
                    )),
+    OpInfo('real',
+           op=torch.real,
+           dtypes=complex_types(),
+           dtypesIfCPU=complex_types(),
+           dtypesIfCUDA=complex_types(),
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_eager'),
```

Comment:
Hey @Kiyosora why do we need these skips? could you share the error message for the failing tests?

Reply:
I think it is because `test_variant_consistency_eager` will fail because real and imag are attributes.

Relevant conversation from the previous PR.

https://github.com/pytorch/pytorch/pull/51946#discussion_r573546348